### PR TITLE
responsive navbar and header

### LINKED ISSUE #62: Responsive Design: Navbar & Header

### PR DESCRIPTION
**Pull request by saifaldin**

### CODE REVIEW

File: frontend/src/components/Navbar/index.jsx [hunk 1]
```diff
@@ -17,42 +17,72 @@ export const Navbar = () => {
         setError(error.message);
       });
   }, []);
+  const toggleMobileNav = () => {
+    document.getElementById('mobileNav').classList.toggle('hidden');
```
Comment (tramsi) [inline frontend/src/components/Navbar/index.jsx]: It's not a good practice to use getElementById .. Avoid manipulating DOM directly 
React have all the tools you need to achieve the same result and logic

**Review by MAlngaawy — CHANGES_REQUESTED** [review]
please consider these changes
- paragraph & heading width
- the image size
- the buttons width and height
- navbar box-shadow instead of padding

**Review by MAlngaawy — APPROVED** [review]
Very Nice Work Saif
LGTM

**Comment by Nilegfx** [comment]
looking forward to merge this PR 😍

**Pull request merged**